packages/components/Tabs/src/TabsItem.types.ts

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { ViewProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { IFocusable, IPressableState } from '@fluentui-react-native/interactive-hooks';
+import { ITextProps } from '@fluentui-react-native/text';
+import { IconProps } from '@fluentui-react-native/icon';
+import { IButtonProps, IButtonTokens } from '@fluentui-react-native/button';
+
+export const tabsItemName = 'TabsItem';
+
+export interface TabsItemInfo extends IPressableState {
+  /**
+   * Disables the TabsItem.
+   * @default false
+   */
+  disabled?: boolean;
+
+  /**
+   * TabsItem icon.
+   */
+  icon?: boolean;
+
+  /**
+   * TabsItem text.
+   */
+  headerText?: boolean;
+
+  /**
+   * Indicates if TabsItem is selected.
+   */
+  selected: boolean;
```

Comment:
Same comment as the one above

Reply:
Selected seems to be needed to be kept in the state for styling. If we remove it from the state, when there is no focus on the control, the selected styling disappears.